src/HashedSimplify.hs

```diff
@@ -24,66 +24,60 @@ import HashedPattern
 import HashedPrettify
 import HashedUtils
 import Prelude hiding
-    ( (*)
-    , (+)
-    , (-)
-    , (/)
-    , (^)
-    , acos
-    , acosh
-    , asin
-    , asinh
-    , atan
-    , atanh
-    , const
-    , const
-    , cos
-    , cosh
-    , exp
-    , log
-    , negate
-    , product
-    , sin
-    , sinh
-    , sqrt
-    , sum
-    , tan
-    , tanh
-    )
+  ( (*)
+  , (+)
+  , (-)
+  , (/)
+  , (^)
+  , acos
+  , acosh
+  , asin
+  , asinh
+  , atan
+  , atanh
+  , const
+  , const
+  , cos
+  , cosh
+  , exp
+  , log
+  , negate
+  , product
+  , sin
+  , sinh
+  , sqrt
+  , sum
+  , tan
+  , tanh
+  )
 import qualified Prelude
 
 -- | For debugging a single simplification rule
 --
-makeTrans ::
-       (DimensionType d, ElementType et)
-    => Transformation
-    -> Expression d et
-    -> Expression d et
+makeTrans :: (DimensionType d, ElementType et) => Transformation -> Expression d et -> Expression d et
 makeTrans smp = wrap . smp . unwrap
 
 -- | Simplify an expression
 --
-simplify ::
-       (DimensionType d, ElementType et) => Expression d et -> Expression d et
+simplify :: (DimensionType d, ElementType et) => Expression d et -> Expression d et
 simplify = wrap . applyRules . unwrap
   where
     applyRules =
-        multipleTimes 100 . chain $
-        [ toRecursiveSimplification evaluateIfPossibleRules
-        , toRecursiveSimplification groupConstantsRules
-        , toRecursiveSimplification combineTermsRules
-        , toRecursiveSimplification combineTermsRulesProd
-        , toRecursiveSimplification powerProdRules
-        , toRecursiveSimplification powerScaleRules
-        , toRecursiveSimplification combinePowerRules
-        , toRecursiveSimplification powerSumRealImagRules
-        , toRecursiveSimplification combineRealScalarRules
-        , toRecursiveSimplification flattenSumProdRules
-        , toRecursiveSimplification reduceSumProdRules
-        . toRecursiveSimplification combineRotateRules
-        , rulesFromPattern
-        , removeUnreachable
-        ]
+      multipleTimes 100 . chain $
```

Comment:
It seems like you're ran `hindent` to this file with 2 spaces instead of 4.
Did you try to run `make format`?